requirements_full.txt

```diff
@@ -8,6 +8,7 @@ numpy>=1.17.0; python_version<"3.12"
 python-dateutil
 PyQt6
 regex
+setuptools
```

Comment:
Oof, is that avoidable ? I don't see what changed in this MR to warrant that, si it means we already were depending on setuptools ?

Reply:
Exactly. Uninstall setuptools, run `pytest -k namespace`, see:
```py
tests/test_manager.py:124: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import warnings
    
    with warnings.catch_warnings():
        warnings.simplefilter("ignore", DeprecationWarning)
>       __import__("pkg_resources").declare_namespace(__name__)
E       AttributeError: module 'pkg_resources' has no attribute 'declare_namespace'

tests/testdata/python3/data/path_pkg_resources_1/package/__init__.py:5: AttributeError
===================================== short test summary info ======================================
FAILED tests/test_manager.py::AstroidManagerTest::test_identify_old_namespace_package_protocol - AttributeError: module 'pkg_resources' has no attribute 'declare_namespace'
========================== 1 failed, 10 passed, 1675 deselected in 1.18s ===========================
```

We could do this in a separate MR if you like, but I figured a separate commit was enough.

Reply:
Isn't this a test dependencie and not a runtime dependencie ?

Reply:
I think that's what this file is.